api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/dsl/GraphTrainableModelBuilderTest.kt

```diff
@@ -19,36 +20,41 @@ internal class GraphTrainableModelBuilderTest {
                     +Input(128, 128)
                     +Conv1D()
                 }
+            }.let {
+                it.layers.size == 2 && it.layers[0] is Input && it.layers[1] is Conv1D
             }
-        }
+        )
 
     }
 
     @Test
     fun buildFunctional() {
-        assertDoesNotThrow {
+        assert(
             functional {
                 model {
                     name = "Test Functional"
                 }
 
                 layers {
                     +Input(128, 128)
-                    +Conv1D()
```

Comment:
https://github.com/JetBrains/KotlinDL/issues/314
Temporary workaround to make test pass